src/kudu/tserver/consensus_service.cc

```diff
@@ -439,13 +439,14 @@ void ConsensusServiceImpl::RunLeaderElection(const RunLeaderElectionRequestPB* r
     s = consensus->StartElection(
         consensus::ElectionMode::ELECT_EVEN_IF_LEADER_IS_ALIVE,
         {consensus::ElectionReason::EXTERNAL_REQUEST,
-         std::move(request_start),
+         request_start,
```

Comment:
Comment for Line 438 above: original_start_time is not documented to be in nanoseconds in the protobuf.

Reply:
Actually let me document the protobuf instead.

Reply:
Ok, it actually is specified in consensus.proto: `Should be specified as nanoseconds since epoch`, I'll just comment here.